go/vt/sqlparser/ast.go

```diff
@@ -4531,6 +4532,28 @@ func (node *SubstrExpr) walkSubtree(visit Visit) error {
 	)
 }
 
+type TrimExpr struct {
+	Str     Expr
+	Pattern Expr
+	Dir     Expr
```

Comment:
Should make this a string and define const values for it (LEADING, TRAILING, BOTH)